README.md

```diff
@@ -9,9 +9,10 @@ When you build plugins for Fastify and you want that them to be accessible in th
 2. Use this module
 
 #### Usage
-`fastify-plugin` can do two things for you:
+`fastify-plugin` can do three things for you:
 - Add the `skip-override` hidden property
 - Check the bare-minimum version of Fastify
+- Ass some custom meta to the plugin
```

Comment:
How about we lose the "Ass" and go with this instead:

> Asses plugin metadata

Reply:
Oooops :D